jupyterlab/error.html

```diff
@@ -0,0 +1,58 @@
+<!--
+Copyright (c) Jupyter Development Team.
+Distributed under the terms of the Modified BSD License.
+-->
+
+<!DOCTYPE HTML>
```

Comment:
1. In general, it's considered best practice to put `<!DOCTYPE html>` as the *very first thing* in an HTML document, before any comments. I'd suggest putting the copyright header after the doc type.
1. I would suggest making `HTML` lowercase. This is just common convention and not an error at all. The `DOCTYPE` itself could even be lowercase, but XML parsers want it to be uppercase, so the convention, generally is to write: `<!DOCTYPE html>` for HTML5 documents.

In fact, I'd make these suggestions for the other HTML files we have as well. Neither of these suggestions are errors outright, just good rules of thumb: http://stackoverflow.com/a/4897850